mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```
@@ -354,7 +352,19 @@ LogicalResult ConvConverter<ConvType>::matchAndRewrite(
   }
 
   int64_t group = op.getGroup();
-  auto accType = op.getAccTypeAttr();
+  auto accType = op.getAccTypeAttr().getValue();
+  // Convert acc_type to a type supported by TOSA Conv2D operation.
+  // F8 types are converted to f16, bf16 is converted to f32, and 8-bit integers are converted to i32.
```

I think we shouldn't add acc_type to migraphx dialect, unless we agree migraphx team needs it. And if we decide to add it. I'd prefer to fail rather than silently changing the acc_type they request.